tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md

```diff
@@ -266,7 +266,6 @@ Execute the following command to run the build:
     gcloud container builds submit --tag=gcr.io/${PROJECT_ID}/hello:v1 .
 
 Replace `${PROJECT_ID}` with the ID of your Google Cloud Platform project.
-The period at the end is required.
```

Comment:
Should we then remove the period in the example command?

Reply:
I debated that. I decided not to because it's like the docker command above, it just happens to default to "." But putting it there is more explicit.

Reply:
sgtm